### PR TITLE
Specification of scrollWidth/scrollHeight under Internet Explorer

### PR DESCRIPTION
**Pull request by aleen42**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

> To specify that it is not unknown feature around `Element.scrollWidth` under Internet Explorer.

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

> There may be exception according to https://stackoverflow.com/questions/30900154/workaround-for-issue-with-ie-scrollwidth

- [x] Data: if you tested something, describe how you tested with details like browser and version

> I have tested from Internet Explorer 5 through 7 via a simualted environment under IE11.

- [x] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any

> #5491

### CODE REVIEW

**Comment by aleen42** [comment]
As I don't have virtual machines for testing through IE 5 to 7, so I have temporarily modified the `version_added` to `8`. The following is the test cases through IE 8 to 11 with the example:

### Case 1

```html
<div id="test" style="width: 100px; height: 100px; padding: 0 20px;"></div>
<script type="text/javascript">
    window.onload = function () {
        console.log(document.getElementById('test').scrollWidth);
    };
</script>
```

| browsers | IE8 | IE9 | IE10 | IE11 |
|:---------|:----|:----|:-----|:-----|
| output   | 40  | 40  | 140  | 140  |

### Case 2

```html
<div id="test" style="width: 100px; height: 100px; padding: 0 20px; box-sizing: border-box"></div>
<script type="text/javascript">
    window.onload = function () {
        console.log(document.getElementById('test').scrollWidth);
    };
</script>
```

| browsers | IE8 | IE9 | IE10 | IE11 |
|:---------|:----|:----|:-----|:-----|
| output   | 40  | 40  | 100  | 100  |

File: api/Element.json [hunk 1]
```diff
@@ -7076,7 +7076,8 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "8",
+              "notes": "The value of <code>scrollWidth</code> affected by padding value from Internet Explorer 8 through 9."
```
Comment (vinyldarkscratch) [inline api/Element.json]: I think we can word this better:

```suggestion
              "notes": "In Internet Explorer 5 through 7, if padding is set, the value of <code>scrollWidth</code> is equal to the sum of the left and right padding.  This behavior was fixed in Internet Explorer 8."
```

File: api/Element.json [hunk 2]
```diff
@@ -7076,7 +7076,8 @@
               "version_added": true
             },
             "ie": {
-              "version_added": null
+              "version_added": "8",
```
Comment (vinyldarkscratch) [inline api/Element.json]:
```suggestion
              "version_added": "5",
```

File: api/Element.json (hunk 1 above)
Comment (aleen42) [reply api/Element.json]: Have you specified the DOCTYPE, I did not specify it in my test

Comment (vinyldarkscratch) [reply api/Element.json]: I pretty much never do when performing tests; it’s kind of like adding a flag to the top of a plain text file saying it’s text, when it’s already pretty obvious.  😛

**Comment by aleen42** [comment]
@vinyldarkscratch  Does the MDN document automatically rebuild according to the database? And what is the timing?

**Comment by vinyldarkscratch** [comment]
BCD releases usually happen weekly (or bi-weekly during holidays and major events) on Thursdays, and then new versions of Kumascript get released the day after.  These changes should show up in the MDN web docs within a week.  😉

**Comment by aleen42** [comment]
👌 Thx.

**Pull request merged**